package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Is this an `npm` generated file? I would have expected it to put things in the `ui` directory, not at the top-level.

Reply:
Hmm, I might have accidentally run `npm` in top-level directory. Deleted this file.